apiserver/Makefile

```diff
@@ -164,13 +164,22 @@ docker-push: ## Push image for the api server.
 
 ##@ Deployment
 .PHONY: install
-install: kustomize ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
-	cd deploy/local && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG)
-	$(KUSTOMIZE) build deploy/local | kubectl create -f -
+install: kustomize docker-image load-image  ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	cd deploy/local/insecure && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG)
+	$(KUSTOMIZE) build deploy/local/insecure | kubectl create -f -
 
 .PHONY: uninstall
 uninstall: ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build deploy/local | kubectl delete -f -
+	$(KUSTOMIZE) build deploy/local/insecure | kubectl delete -f -
+
+.PHONY: install-secure
+install-secure: kustomize docker-image security-proxy-image load-image load-security-proxy-image  ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
+	cd deploy/local/secure && $(KUSTOMIZE) edit set image kuberay/apiserver=$(IMG) && $(KUSTOMIZE) edit set image kuberay/security-proxy=kuberay/security-proxy:latest
+	$(KUSTOMIZE) build deploy/local/secure | kubectl create -f -
+
+.PHONY: uninstall-secure
+uninstall-secure: ## Install the kuberay api server to the K8s cluster specified in ~/.kube/config.
```

Comment:
you might want to fix the descrition of the target also.  if you type `make help` you should see the targets and a description. If the `## some text` is missing it will not show up in the output of the `make help` target.

Reply:
Done